thefuck/rules/chdir.py

```diff
@@ -0,0 +1,26 @@
+import re
```

Comment:
This file is very similar to the cd variant and so we have duplicated logic. Do we think it's possibl/worthwhile combining them?